sys/cheri/cherireg.h

```diff
@@ -116,8 +116,13 @@
 #define	CHERI_SEALCAP_KERNEL_OFFSET	0x0
 
 /* Reserved CHERI object types: */
+#ifdef __aarch64__
+#define	CHERI_OTYPE_UNSEALED	(0)
+#define	CHERI_OTYPE_SENTRY	(-1l)
```

Comment:
This is wrong, but is correct on morello-dev and in #842, so something went wrong in extracting the diff. Given kernel support should land before hybrid userspace due to the dependency on kernel headers I will drop this diff entirely from this PR and leave it to #842 where it's (a) correct (b) puts the macros back in MD headers where they belong.

Reply:
(now moved into #740 due to that also needing the right otype definitions)